src/ontology/metadata/mondo.sssom.config.yml

```diff
@@ -76,6 +76,274 @@ curie_map:
   # SNOMEDCT_2010_1_31: "http://purl.obolibrary.org/obo/mondo/mappings/unknown_prefix/SNOMEDCT_2010_1_31/"
   OMIA: "http://purl.obolibrary.org/obo/mondo/mappings/unknown_prefix/OMIA/"
 
+extended_prefix_map:
```

Comment:
## Check / update for correctness @joeflack4 
- [ ] Check / update for correctness

<details><summary>Details</summary>
<p>

This `extended_prefix_map` was created by transforming the `curie_map` in `mondo.sssom.config.yml` in `mondo-ingest`. However, that `curie_map` is not the same as the one here. I need to recreate this `extended_prefix_map` so that it is the most up-to-date. I don't know if (a) there were things about `mondo-ingest`'s version that were better than `mondo`, or (b) there were things about the `mondo` version that were better than `mondo-ingest`, or (c) both. I suspect 'b', but I'd have to look into it. I'm also not sure I'm the best person to make that determination.

To recreate the EPM, I could use the new `curies` function that Charlie and I created to generate an EPM based on an input bimap (e.g. `curie_map` in `mondo.sssom.config.yml`). Or I could just create it manually.

</p>
</details>

Reply:
The EPM looks great but as I said above, we probably need it only in mondo-ingest if we sever the connection between mondo.sssom.config in Mondo and mondo ingest repo due to their very different use cases (mondo ingest: ETL, lexmatch; mondo repo: publication of SSSOM files to community.

Reply:
Ahh, OK. I'm resolving this thread now then. See action items here: https://github.com/monarch-initiative/mondo/pull/7199#discussion_r1467092075